SwrveSDK/SDK/Conversation/SwrveConversationCampaign.h

```diff
@@ -40,9 +41,10 @@
  * \returns Message setup for the given trigger or nil.
  */
 -(SwrveConversation*)getConversationForEvent:(NSString*)event
-                        withAssets:(NSSet*)assets
-                            atTime:(NSDate*)time
-                       withReasons:(NSMutableDictionary*)campaignReasons;
+                              withParameters:(NSDictionary*)parameters
```

Comment:
Payload or parameters? Also, same as above, do we need to keep old and new variations of this method?